src/Data/Form.purs

```diff
@@ -223,6 +223,15 @@ noErrors (Errors e) = Data.Set.isEmpty e
 errorsToArray :: ∀ e. Ord e => Errors e -> Array e
 errorsToArray (Errors e) = toUnfoldable e
 
+errors :: forall a. Field a -> Errors ValidationError
```

Comment:
This function exists because the `errors` field is within each `Input _`. Should we move this up to the top level, or should we leave it where it is as we plan to have errors specific to each case?

This is more a general question that we should address in a separate PR if there's something to address.

Reply:
Hmm, yeah I guess we need to circle back and figure out how we want to model errors. That might be something to block out sprint work for so that we actually have time to focus on it.